_tasks/dnt.ts

```diff
@@ -50,5 +52,5 @@ await Promise.all([
     typeCheck: false,
   }),
   fs.copy("LICENSE", path.join(outDir, "LICENSE")),
-  fs.copy("Readme.md", path.join(outDir, "Readme.md")),
+  fs.copy("help.txt", path.join(outDir, "help.txt")),
```

Comment:
We should probably still copy the readme

Reply:
If we do, it'll contain dead links (given its in a different directory)

Reply:
Better to have a readme with dead links than no readme at all, IMO